File: codes/android/smartqApp/src/org/sharp/vocreader/beans/Course.java
package org.sharp.vocreader.beans;

import org.sharp.vocreader.intf.Constants;


public class Course {
	public int courseNo;
	public int status;
	public int zipSize;
	
	public Course(){
	}
	
	public Course(int courseNo){
		this.courseNo = courseNo;
		this.status = Constants.COURSE_STATUS_NOTEXIST;
	}
	
	public Course(Course course){
		this.courseNo = course.courseNo;
		this.status = course.status;
		this.zipSize = course.zipSize;
	}
	
	public String toString(){
		if(status == Constants.COURSE_STATUS_NOTEXIST)
			return "Download Course "+courseNo;
		else if(status == Constants.COURSE_STATUS_DOWNLOADED)
			return "Open Course "+courseNo;
		else
			return "Course "+courseNo;
	}

	public int getStatus() {
		return status;
	}

	public void setStatus(int status) {
		this.status = status;
	}
}

File: codes/os-x/prjs/jl/jl/core/data/Course.cpp
#include "Course.h"

//const int Course::STATUS_DOWNLOADED = 0;
//const int Course::STATUS_NOTEXIST_NOINFO = 1;
//const int Course::STATUS_NOTEXIST = 2;

Course::Course()
{
	//ctor
}

Course::~Course()
{
	//dtor
}


int Course::getCourseNo(){
	return courseNo;
}
void Course::setCourseNo(int courseNo){
	this->courseNo = courseNo;
}
//int Course::getStatus(){
//	return status;
//}
//void Course::setStatus(int status){
//	this->status = status;
//}
//int Course::getZipSize(){
//	return zipSize;
//}
//void Course::setZipSize(int zipSize){
//	this->zipSize = zipSize;
//}
